mandatory/2-function-creation.js

```diff
@@ -15,26 +21,60 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    return 0.9 * totalPrice;
+  } else if (totalPrice > 0 && totalPrice < 200) {
+    return 0.95 * totalPrice;
+  } else {
+    return 0
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+
+
+  function printOddNumbers(limit) {
+    let i = 0;
+    while ( i <= limit) {
+      if (i % 2 === 1) {
+        console.log(i);
+      } 
+      i = i + 1;
+    }
+  }
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 > price2) {
+    return price1;
+  } else if (price2 > price1) {
+    return price2;
+  } else price1;
+}
 
```

Comment:
well done Afsha .

Reply:
Thank you Navid!

Reply:
Agreed, good job 👍 
It's good that you're thinking about the case where `price1` is equal to `price2`.